src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+{
+
+	"death.attack.mmorpg": "%1$s был убит %2$s",
+	"block.mmorpg.harvest.purple": "Хаотические семена",
+	"block.mmorpg.harvest.green": "Первозданные семена",
+	"block.mmorpg.harvest.blue": "Прозрачные семена",
+	"item.mmorpg.custom_loot_crate": "Ящик с добычей",
+	"item.mmorpg.loot_crate": "Ящик с добычей",
+	"item.mmorpg.custom_crate": "Ящик с экипировкой",
+
+"_comment": "# CREATIVE TAB",
+
+	"itemGroup.mmorpg.tab": "Mine and Slash",
+
+	"death.attack.mmorpg.custom_damage": "Много урона!",
+	"death.attack.death_potion": "%1$s подумал, что это отличная идея выпить зелье с названием 'Смерть'.",
+
+"_comment": "STATIONS",
+
+	"block.mmorpg.teleporter": "Картографическое устройство",
+
+"_comment": "ITEM TOOLTIPS",
+
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
+	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
+	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
+	"mmorpg.tooltip.capacitor2": "Множитель топлива",
+	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо слота",
+
+"_comment": "RARITY ITEM NAMES",
+
+"_comment": "MISC ITEMS",
+
+	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
+	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
+	"item.mmorpg.jewel.int": "Звездная пыль",
+	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет Рунной мастерской",
+	"item.mmorpg.stone.0": "Обычный камень",
+	"item.mmorpg.stone.1": "Необычный камень",
+	"item.mmorpg.stone.2": "Редкий камень",
+	"item.mmorpg.stone.3": "Эпический камень",
+	"item.mmorpg.stone.4": "Легендарный камень",
+	"item.mmorpg.stone.5": "Мифический камень",
+	"item.mmorpg.map": "Карта приключений",
+	"item.mmorpg.stat_soul": "Душа характеристики снаряжения",
+	"item.mmorpg.set_map": "Карта установки",
+	"item.mmorpg.loot_bag": "Сумка добычи снаряжения",
+	"item.mmorpg.map_bag": "Сумка карт",
+	"item.mmorpg.currency_bag": "Сумка валюты",
+	"item.mmorpg.newbie_gear_bag": "Сумка новичка",
+	"item.mmorpg.tutorial_book": "Учебник эпохи изгнания",
+	"item.mmorpg.map_back_portal": "Жемчуг отхода с карты",
```

Тут я сделал вывод что ты скорее всего сам не читал свой перевод, но если просто не знаешь английского, то тогда вопрос снимается (но тогда надо вдвойне ответственно подходить к этому вопросу)

Изменения тут оставляю на твою совесть
```suggestion
	"item.mmorpg.map_back_portal": "Жемчуг отхода с карты",
```

---

С анлиским на самом деле не всё так хорошо, в следующих тестах решу нужны ли здесь правки